specification/elasticsan/resource-manager/Microsoft.ElasticSan/preview/2024-07-01-preview/elasticsan.json

```diff
@@ -2422,234 +2755,335 @@
+          "description": "Private Link Service Connection State."
         },
-        "provisioningState": {
-          "description": "State of the operation on the resource.",
-          "$ref": "#/definitions/ProvisioningState",
-          "readOnly": true
+        "groupIds": {
+          "type": "array",
+          "description": "List of resources private endpoint is mapped",
+          "items": {
+            "type": "string"
+          }
         }
+      },
+      "required": [
+        "privateLinkServiceConnectionState"
+      ]
+    },
+    "PrivateEndpointServiceConnectionStatus": {
+      "type": "string",
+      "description": "The private endpoint connection status.",
+      "enum": [
+        "Pending",
+        "Approved",
+        "Failed",
+        "Rejected"
+      ],
+      "x-ms-enum": {
+        "name": "PrivateEndpointServiceConnectionStatus",
+        "modelAsString": true,
+        "values": [
+          {
+            "name": "Pending",
+            "value": "Pending"
+          },
+          {
+            "name": "Approved",
+            "value": "Approved"
+          },
+          {
+            "name": "Failed",
+            "value": "Failed"
+          },
+          {
+            "name": "Rejected",
+            "value": "Rejected"
+          }
+        ]
       }
     },
-    "ManagedByInfo": {
+    "PrivateLinkResource": {
       "type": "object",
-      "description": "Parent resource information.",
+      "description": "A private link resource",
       "properties": {
-        "resourceId": {
-          "type": "string",
-          "x-ms-client-name": "ResourceId",
-          "description": "Resource ID of the resource managing the volume, this is a restricted field and can only be set for internal use."
+        "properties": {
+          "$ref": "#/definitions/PrivateLinkResourceProperties",
+          "description": "Resource properties.",
+          "x-ms-client-flatten": true
         }
-      }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/Resource"
+        }
+      ]
     },
-    "VolumeUpdate": {
+    "PrivateLinkResourceListResult": {
       "type": "object",
-      "description": "Response for Volume request.",
+      "description": "A list of private link resources",
       "properties": {
-        "properties": {
-          "x-ms-client-flatten": true,
-          "description": "Properties of Volume.",
-          "$ref": "#/definitions/VolumeUpdateProperties"
+        "value": {
+          "type": "array",
+          "description": "Array of private link resources",
+          "items": {
+            "$ref": "#/definitions/PrivateLinkResource"
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "URI to fetch the next section of the paginated response.",
+          "readOnly": true
         }
-      }
+      },
+      "required": [
+        "value"
+      ]
```

Comment:
`value` should be optional for `PrivateLinkResourceListResult`
<img width="704" height="31" alt="image" src="https://github.com/user-attachments/assets/36ca66a4-02df-40d4-a9f8-fa762ae591d6" />